### PR TITLE
Taking out Hertz and Seconds units along with general Julia 1.5 updates

### PR DESCRIPTION
**Pull request by jparcill**
- this commit was aiming for the bare minimum of being able to run the `MusicProcessing.jl` file without any errors

### CODE REVIEW

File: src/MusicProcessing.jl [hunk 1]
```diff
@@ -1,19 +1,14 @@
...
-typealias PCM16Sample Fixed{Int16, 15}
-typealias PCM32Sample Fixed{Int32, 31}
-
-# shortcut to SIQuantity type of frequency and time
-typealias Hertz SIUnits.SIQuantity{Int64,0,0,-1,0,0,0,0,0,0}
-typealias Seconds{R} SIUnits.SIQuantity{R,0,0,1,0,0,0,0,0,0}
+const PCM16Sample = Fixed{Int16, 15}
+const PCM32Sample = Fixed{Int32, 31}
 
 export Hz, kHz, s, ..
```
Comment (Datseris) [inline src/MusicProcessing.jl:16]: `export Hz, kHz, s,` are still exported? What brings them into scope?

**Comment by Datseris** [comment]
@jparcill thanks, I've noticed that this package has no tests whatsoever... Can you add something in the `runtests.jl` file? Perhaps we can add a small mp3 file in the test folder.

File: src/MusicProcessing.jl (hunk 1 above)
Comment (jparcill) [reply src/MusicProcessing.jl:16]: Whoops I probably still need to remove them. But I believe that SampledSignals brings them into scope

**Comment by Datseris** [comment]
@jparcill sorry, I kind of lost track here. I need to spend some time reading the source code in detail. Meanwhile, can you tell me the following: Why do we have to remove the SampledSignal units? The package uses SampledSignals anyway, right...?

**Comment by jparcill** [comment]
So the units `Hertz` and `Seconds` that I replaced with Real throughout the code is from `SIUnits` which is deprecated.

SampledSignals exports `Hz`, `kHz`, and `s`: these are actually from Unitful as you can see in the README: https://github.com/JuliaAudio/SampledSignals.jl

If we want to actually replace `Hertz` and `Seconds` with `Hz` and `s` we would have to import Unitful for typechecking. Is that the direction we want to go in?

**Comment by Datseris** [comment]
By now you probably are more familiar with the source than me, so you should make the decision:

If the units `Hz, s` are actually used, and actually useful in the source (e.g. does `SampledSignals` already return arrays with Units?), then we just replace the old stuff with the new stuff by re-using what `SampledSignals` provides, which already is `Hz, s`. 

If units are in general not used anywhere in the source, then we can remove them.

**Comment by jparcill** [comment]
Maybe for consistency's sake it makes sense to keep it in. The source itself doesn't use the units in any way different than using a Float, however it seems that `SampleBuf`(the audio type is used from SampleSignals) allows for indexing using `Hz` or `s`

**Comment by jparcill** [comment]
For this commit, I just put back Seconds and Hertz in functions where it made sense.

In the other cases, Hertz was used for SampleBuf which it seems can't be done anymore.

**Comment by Datseris** [comment]
Alright! Do we have any tests anywhere? Is the code runnable now?

**Comment by jparcill** [comment]
Nope haven't tested anything yet, just replacing old code syntax with new.

I'll have those as my next todos

**Comment by Datseris** [comment]
Okay, I'll let you add tests in a separate PR!

**Pull request merged**